docs/pipelineruns.md

```diff
@@ -156,6 +140,27 @@ spec:
        ...
 ```
 
+#### Tekton Bundles
```

Comment:
can we add a link to this new section in the TOC at the top?

Reply:
done :wink: